pythonforandroid/recipes/sdl2_mixer/toggle_modplug_mikmod_smpeg_ogg.patch

```diff
@@ -1,29 +1,28 @@
---- orig/Android.mk	2016-01-03 07:15:57.000000000 +0100
-+++ patch/Android.mk	2016-04-15 21:28:55.169697882 +0200
-@@ -6,22 +6,22 @@
- 
- # Enable this if you want to support loading MOD music via modplug
- # The library path should be a relative path to this directory.
--SUPPORT_MOD_MODPLUG ?= true
-+SUPPORT_MOD_MODPLUG := false
- MODPLUG_LIBRARY_PATH := external/libmodplug-0.8.8.4
- 
- # Enable this if you want to support loading MOD music via mikmod
- # The library path should be a relative path to this directory.
--SUPPORT_MOD_MIKMOD ?= true
-+SUPPORT_MOD_MIKMOD := false
```

Comment:
libmikmod also appears to be LGPL. It would therefore also be reasonable not to be included by default, but your pull request seems to remove the `SUPPORT_MOD_MIKMOD := false`. Is that intentional?

Reply:
SUPPORT_MOD_MIKMOD flag is not available anymore cause mikmod library is been removed since sdl2_mixer release 2.0.2

Reply:
right, another one of the removed ones I missed, makes sense :+1: thanks for the info